packages/cra-template-mephisto-review/template.json

```diff
@@ -0,0 +1,19 @@
+{
+  "package": {
+    "dependencies": {
+      "mephisto-review-hook": "^1.0.3",
+      "@testing-library/jest-dom": "^5.11.4",
+      "@testing-library/react": "^11.1.0",
+      "@testing-library/user-event": "^12.1.10",
+      "web-vitals": "^0.2.4"
```

Comment:
I'm thinking we can remove these last 4 dependencies here since we aren't really using them at the moment. Keep the package lean.